src/components/app-navigator/TopBar.tsx

```diff
@@ -82,28 +97,38 @@ export const TopBar = memo(
     return (
       <View style={styles.root}>
         <View style={styles.topBar}>
-          {back ? (
-            <TopBarArrowBack onPress={topBarNavigation.goBack} />
-          ) : (
-            <IconButton
-              icon={IconNotification}
-              styles={{ icon: styles.iconNotification }}
-              fill={neutralLight4}
-              onPress={handlePressNotification}
-            />
-          )}
+          <View style={styles.headerLeft}>
+            {headerLeft ? (
+              headerLeft({})
```

Comment:
What is the type of the args here out of curiousity?

Reply:
oh yeah good call, its like "touchTint", "color", basically things to help the styles of the button when it's clicked. thinking for now we can ignore it, since these items should be buttons anyway that can handle themselves